packages/denylist-bot/src/services/consumer.ts

```diff
@@ -0,0 +1,91 @@
+import type { Env } from "../types/env";
+import type { QueuedData } from "../types/pagination";
+import { setupNear, viewFunction, AttachedGas } from "../utils";
+
+const riskMap: any = {
+  "Scam": 5,
+  "Theft": 5,
+  "Sanctions": 5,
+  "ChildAbuse": 5,
+  "Ransomware": 5,
+  "Counterfeit": 5,
+  "DarknetService": 5,
+  "TerroristFinancing": 5,
+  "IllicitOrganization": 5,
+};
+
+/** fetchRiskParams
+ * This function must perform a view call to contract to fetch
+ * there is currently no such view call in the contract. Mock
+ * @param category The risk category
+ * @returns Returns the max risk accepted by the contract
+ */
+const fetchRiskParams = async (category: string): Promise<number> => {
+  return riskMap[category];
+}
+
+/** sendBanTransaction
+ * This function must sign and send a denylist transaction to the contract
+ * @param payload The queue payload
+ * @param env The env object
+ * @returns void
+ */
+const sendBanTransaction = async (payload: QueuedData, env: Env): Promise<void> => {
+  const connection = await setupNear(env);
+
+  const account = await connection.account(env.ACCOUNT_ID);
+
+  await account.functionCall({
+    contractId: env.HYC_CONTRACT,
+    methodName: "denylist",
+    args: {
+      account_id: payload.account,
+    },
+    gas: AttachedGas as any,
+    attachedDeposit: "320000000000000000000" as any,
```

Comment:
I'd make this attached deposit a constant defined on top of the file or in constants file to improve readability